core/arch/x86/decode_table.c

```diff
@@ -7155,6 +7189,30 @@ const instr_info_t evex_W_extensions[][2] = {
   }, { /* evex_W_ext 99 */
     {OP_vpermt2ps,0x66387f18,"vpermt2ps",Ve,xx,KEw,He,We,mrm|evex|reqp,x,END_LIST},
     {OP_vpermt2pd,0x66387f58,"vpermt2pd",Ve,xx,KEb,He,We,mrm|evex|reqp,x,END_LIST},
+  }, { /* evex_W_ext 100 */
+    {OP_vextractf32x4, 0x663a1918, "vextractf32x4", Wdq, xx, KE4b, Vdq_e, Ib, mrm|evex|reqp, x, END_LIST},
```

Comment:
See comment above: we always put immeds as the first source operand.

Reply:
Fixed, see above. Note that to be consistent, I have changed the definition of the VEX encoded versions of vinsertps and vextractps as well, and have added this to the release notes.